widgetastic_manageiq/vm_reconfigure.py

```diff
@@ -30,7 +31,8 @@ class DisksTable(VanillaTable):
         # Workaround necessary until BZ 1524960 is resolved
         4: BootstrapSelect(id='hdUnit'),
         3: BootstrapSelect(id='hdUnit'),
-        'Dependent': BootstrapSwitch(name='cb_dependent'),
+        'Dependent': BootstrapSwitch(
```

Comment:
@vatsalparekh, this issue should disappear when the following my PR gets merged https://github.com/RedHatQE/widgetastic.core/pull/68

Reply:
You mean the issue that the `VersionPick` not getting resolved would disappear when that PR is merged, right?

Reply:
exactly